posthog/models/feature_flag.py

```diff
@@ -91,6 +114,21 @@ def is_group_match(self, group: Dict, group_index: int):
     def _match_distinct_id(self, group_index: int) -> bool:
         return len(self.query_groups) > 0 and self.query_groups[0][group_index]
 
+    # Define contiguous sub-domains within [0, 1].
+    # By looking up a random hash value, you can find the associated variant key.
+    # e.g. the first of two variants with 50% rollout percentage will have value_max: 0.5
+    # and the second will have value_min: 0.5 and value_max: 1.0
+    @property
+    def variant_lookup_table(self):
+        lookup_table = []
+        value_min = 0
+        for variant in self.feature_flag.variants:
```

Comment:
Can you walk me through a scenario?

```
- We have a feature flag: "Home Experiment" that has an 80% roll-out
- We have 3 variants: Variant A, Variant B, Variant C
```

What would acceptable rollout values be for Variant A, B, and C? Should these 3 variant rollouts sum to 100 or should they sum to 80?

Reply:
They should sum to 100. May become clearer after I add tests.